### PR TITLE
PP-1462, PP-1437 Disable Afterpay, Latitude Gem for  AUD, Enable Giropay for DE region

### PR DESCRIPTION
**Pull request by kaustuva-ghosh**
This change is to remove Afterpay and Latitude Gem from AU and add Giropay to DE ( Germany )
This change also removes Hoolah/Shopback from HKD as they no longer support HK

### CODE REVIEW

File: src/index.test.ts [hunk 1]
```diff
@@ -111,18 +111,6 @@ describe("getRegionByCurrency()", function() {
   });
 });
 
-describe("getPaymentMethodsByRegion()", function() {
```
Comment (nicholaspuglia) [inline src/index.test.ts:114]: Why are we removing this test?

Comment (kaustuva-ghosh) [reply src/index.test.ts:114]: This function was added by me, but we don't really use it anywhere

**Review by nicholaspuglia — APPROVED** [review]

File: src/index.test.ts (hunk 1 above)
Comment (kaustuva-ghosh) [reply src/index.test.ts:114]: I've removed the function as well, since it was declared but unused.

**Comment by kaustuva-ghosh** [comment]
Removed Hoolah from HKD as well in the last commit

**Pull request merged**